pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -273,6 +273,11 @@ def use_deepspeed(self) -> bool:
 
     @property
     def is_distributed(self) -> bool:
+        # Used for custom plugins.
+        # Custom plugins should implement is_distributed property.
+        if hasattr(self.training_type_plugin, 'is_distributed') and not \
+                self.on_tpu:
```

Comment:
```suggestion
        if hasattr(self.training_type_plugin, 'is_distributed') and not self.on_tpu:
```
we have 120 line not 80